code/modules/hydroponics/seed_extractor.dm

```diff
@@ -110,16 +110,18 @@
 	var/production = 0
 	var/yield = 0
 	var/potency = 0
+	var/instability = 0
 	var/amount = 0
 
-/datum/seed_pile/New(name, life, endur, matur, prod, yie, poten, am = 1)
+/datum/seed_pile/New(name, life, endur, matur, prod, yie, poten, insta, am = 1)

```

Comment:
always be VERY careful with adding new parameters to procs when not adding them to the end

good idea to check any instancing of seed_pile if you didn't already to make sure this didn't have unintended side effects

(this is how image datums broke months ago when some guy added a new parameter to middle of the constructor)

Reply:
Already checked, and seed_piles are only used by this machine.